docs/advanced_topics/images/feature_detection.rst

```diff
@@ -58,8 +63,53 @@ You can test that OpenCV can be seen by Wagtail by opening up a python shell (wi
 If you don't see an ``ImportError``, it worked. (If you see the error ``libdc1394 error: Failed to initialize libdc1394``, this is harmless and can be ignored.)
 
 
+Rustface
+~~~~~~~~
+
+Rustface_ is python library with prebuilt wheel files provided for Linux and macOS. Although implemented in Rust it is pip-installable:
```

Comment:
is python library -> is a Python library

Reply:
Fixed by 5ea9e81